src/test/mediapipe/python/fail_script.py

```diff
@@ -0,0 +1,32 @@
+#*****************************************************************************
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#*****************************************************************************
```

Comment:
Keep this script, but also add another where initialize method exists, but throws exception that is not caught and handled in Python. (to make sure our try catch block in initialization code in OVMS handles such case)